### PR TITLE
`Transitions`: remove is prefix in props

### LINKED ISSUE #3419: Rename `props`: `is*` eliminates all `is` prefix in the `Transitions` package
package: @yamada-ui/transitions

### Description

Change and unify the names of props.

Removing all is prefix from is*

### Problem Statement/Justification

For example, isOpen becomes open and isDisabled becomes disabled

Unify orientation and direction to either one

props to be changed are marked `@deprecated` in v1

### Proposed Solution or API

_No response_

### Alternatives

_No response_

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

Yes

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by nathanlao**
Closes #3419

## Description

Removing all `is` prefix from `is*`

## Current behavior (updates)

Before: `isOpen`, `isDisabled`, `isReadOnly`

## New behavior

After: `open`, `disabled`, `readOnly`

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Review by taroj1205 — COMMENTED** [review]
Hi @nathanlao, thanks for your PR!
Why do we need to disable these eslint rules?

**Comment by nathanlao** [comment]
Hi @taroj1205, I also want to get clarifications, do we need to ensure the new props take precedence? I got the idea from other [merged PR](https://github.com/yamada-ui/yamada-ui/pull/2265/files#diff-9b2c9827571b959bad43dd78a4f0963064ad78852c069d8cad9ac2b0928e3d39), if they specified both props at the same time, the new one should take precedence. I added this to ignore the warning, but I also don't think this is a good idea to disable the eslint rules like that. Any suggestions are welcome, thank you!

File: packages/components/transitions/src/airy.tsx [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable perfectionist/sort-objects */
```
Comment (hirotomoyamada) [inline packages/components/transitions/src/airy.tsx:1]: Remove this line.

File: packages/components/transitions/src/airy.tsx [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+/* eslint-disable perfectionist/sort-interfaces */
```
Comment (hirotomoyamada) [inline packages/components/transitions/src/airy.tsx:2]: Remove this line.

File: packages/components/transitions/src/collapse.tsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable perfectionist/sort-objects */
```
Comment (hirotomoyamada) [inline packages/components/transitions/src/collapse.tsx:1]: Remove this line.

File: packages/components/transitions/src/flip.tsx [hunk 4]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable perfectionist/sort-objects */
```
Comment (hirotomoyamada) [inline packages/components/transitions/src/flip.tsx:1]: Remove this line.

File: packages/components/transitions/src/flip.tsx [hunk 5]
```diff
@@ -1,3 +1,5 @@
...
+/* eslint-disable perfectionist/sort-interfaces */
```
Comment (hirotomoyamada) [inline packages/components/transitions/src/flip.tsx:2]: Remove this line.

File: packages/components/transitions/src/fade.tsx [hunk 6]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable perfectionist/sort-objects */
```
Comment (hirotomoyamada) [inline packages/components/transitions/src/fade.tsx:1]: Remove this line.

File: packages/components/transitions/src/slide-fade.tsx [hunk 7]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable perfectionist/sort-objects */
```
Comment (hirotomoyamada) [inline packages/components/transitions/src/slide-fade.tsx:1]: Remove this line.

File: packages/components/transitions/src/scale-fade.tsx [hunk 8]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable perfectionist/sort-objects */
```
Comment (hirotomoyamada) [inline packages/components/transitions/src/scale-fade.tsx:1]: Remove this line.

File: packages/components/transitions/src/rotate.tsx [hunk 9]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable perfectionist/sort-objects */
```
Comment (hirotomoyamada) [inline packages/components/transitions/src/rotate.tsx:1]: Remove this line.

File: packages/components/transitions/src/rotate.tsx [hunk 10]
```diff
@@ -1,3 +1,5 @@
...
+/* eslint-disable perfectionist/sort-interfaces */
```
Comment (hirotomoyamada) [inline packages/components/transitions/src/rotate.tsx:2]: Remove this line.

File: packages/components/transitions/src/slide.tsx [hunk 11]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable perfectionist/sort-objects */
```
Comment (hirotomoyamada) [inline packages/components/transitions/src/slide.tsx:1]: Remove this line.

**Comment by nathanlao** [comment]
@hirotomoyamada 
Thank you for confirming, I have removed them. 

Just to clarify, do we need to consider the precedence, say prioritize one prop over the other if both new and disabled props are provided at the same time?

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**